src/index.js

```diff
@@ -0,0 +1,32 @@
+const twit = require('twit');
+require('dotenv').config();
+
+const T = new twit({
+    consumer_key: process.env.CONSUMER_KEY,
+    consumer_secret: process.env.CONSUMER_SECRET,
+    access_token: process.env.ACCESS_TOKEN,
+    access_token_secret: process.env.ACCESS_TOKEN_SECRET
+});
+
+async function search(query) {
+    try {
+        return T.get('search/tweets', {q: query, count: 1});
+    } catch (error) {
+        console.log("Failed to search tweets ", error)
+        throw error;

```

Comment:
Explicitly throwing error here to disrupt the flow and not continue on to retweet if our search fails